android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/viewshot/RNViewShotModule.java

```diff
@@ -33,12 +32,10 @@ public class RNViewShotModule extends ReactContextBaseJavaModule {
     public static final String RNVIEW_SHOT = "RNViewShot";
 
     private final ReactApplicationContext reactContext;
-    private final ScopedContext mScopedContext;
 
-    public RNViewShotModule(ReactApplicationContext reactContext, ScopedContext scopedContext) {
+    public RNViewShotModule(ReactApplicationContext reactContext) {
```

Comment:
This was expo-specific that we're applying in some code transformations in the vendoring script. I believe we still have to keep it, otherwise view shots would be saved in the same directory for all projects opened through Expo Go.

Reply:
Removed this patch